### ISSUE TITLE
Problem with changing extruder

### ISSUE THREAD

**Issue by u4studio** [ZONESTAR3D/Z8#1]
When extruder was changed while printing (part of item one color another part different color) new extruder continues to print with 0 temp..

**Comment by skink32** [comment]
That's right, the change is too short so that no color gradient or color change is visible.

I have already reported this and there should be an update.
Actually this week but I think it will take some time.

**Comment by u4studio** [comment]
> That's right, the change is too short so that no color gradient or color change is visible.
> 
> I have already reported this and there should be an update.
> Actually this week but I think it will take some time.

Sounds good :) Let`s wait..
